db/migrate/20180606173823_add_student_photo_to_student.rb

```diff
@@ -0,0 +1,5 @@
+class AddStudentPhotoToStudent < ActiveRecord::Migration[5.1]
```

Comment:
This migration isn't clear to me and I'd want to remove it. It seems to add a second foreign key pointing in the other direction. As far as I understand we never want to have two foreign keys, because there should only be a single source of truth about what belongs to what.

Reply:
👍 The idea before was to hold two different truths :)